### PR TITLE
refactor(career site): career site color changes

### LINKED ISSUE #179: Career site fixes: colors
Apply these changes:
- set page background color to tblr-white  (#ffffff)
- set recaptcha and footer texts color to tblr-gray-500 (#6c7a91)

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #179.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/career_site/positions/show.html.slim [hunk 1]
```diff
@@ -68,4 +68,4 @@
           size: :medium,
           class: "flex-grow-1 flex-md-grow-0 mt-2" \
         ).with_content(t("career_site.positions.submit_application"))
-      .mt-2.font-small.text-gray-600 = t("recaptcha.privacy_message_html")
+      .mt-2.font-small.text-secondary = t("recaptcha.privacy_message_html")
```
Comment (wrmk) [inline app/views/career_site/positions/show.html.slim:71]: may we use `tblr-gray-500` ?

File: app/assets/stylesheets/src/career_site/_main.scss [hunk 2]
```diff
@@ -36,9 +36,9 @@ footer {
   a,
   span,
   div {
-    color: $gray-600;
+    color: $gray-500;
```
Comment (wrmk) [inline app/assets/stylesheets/src/career_site/_main.scss:39]: Ilya already used `var(--tblr-secondary);` [here](https://github.com/freeats/freeats/pull/166/files)

Comment (wrmk) [reply app/assets/stylesheets/src/career_site/_main.scss:39]: may we unite this?

File: app/views/career_site/positions/show.html.slim (hunk 1 above)
Comment (wrmk) [reply app/views/career_site/positions/show.html.slim:71]: as i see `text-secondary` hangs not `tblr-gray-500`

Comment (ksnitsky) [reply app/views/career_site/positions/show.html.slim:71]: text-secondary is gray-500

Comment (ksnitsky) [reply app/views/career_site/positions/show.html.slim:71]: And there's no text-gray-500 class

**Review by wrmk — APPROVED** [review]

**Pull request merged**